src/components/date-picker/date-picker.tsx

```diff
@@ -182,11 +182,12 @@ export class DatePicker {
             onAction: this.clearValue,
         };
 
-        if (this.useNative) {
+        if (this.disabled || this.readonly || this.useNative) {
```

Comment:
Hmm… this doesn't work, it's not showing the existing value… 🤔

https://user-images.githubusercontent.com/154725/121667137-44a75480-caaa-11eb-8ba6-0ff6bd2e72d4.mov

![image](https://user-images.githubusercontent.com/154725/121667206-52f57080-caaa-11eb-9fad-acdb5e07c62e.png)

Interestingly, it works on mobile (tested both Android and iOS) 😮

![image](https://user-images.githubusercontent.com/154725/121667508-a667be80-caaa-11eb-9775-a1ace6ec5cb9.png)

So, I guess it's something about using the native picker on desktop, or at least in Chrome on Mac.